Aegis Station

JasperStingtail opened the pull request:
# Description

New map: Aegis station- GusStation- is real now. (Hopefully not another Ohio)
---

# Tasks

- [x] Map is made

---

<details><summary><h1>Media</h1></summary>
<p>

<!-- https://cdn.discordapp.com/attachments/809937496279613460/1198147939101581462/image.png -->

![Content Client_ZHgFuxYT40](https://github.com/Simple-Station/Parkstation/assets/77995199/48e7ea04-f175-4de4-83f4-d0d4c5de8f12)


</p>
</details>

---

# Changelog

:cl:
- add: Added Aegis station/GusStation!

JasperStingtail committed:
first commit map and prototype

JasperStingtail committed:
removed nuke (temp), added small OR, named station Aegis

JasperStingtail committed:
dorms, new nuke, court

JasperStingtail committed:
fixed id in map file

JasperStingtail committed:
i dont remember what i did, more progress

JasperStingtail committed:
security, hop

JasperStingtail committed:
sec stuff

JasperStingtail committed:
rigged and piped most of atmos, still needs to be cleaned up

JasperStingtail committed:
finished product

JasperStingtail committed:
fixed air alarms, grilles

JasperStingtail committed:
extinguishers, minor fixes, defib fixes, air alrm fixes

JasperStingtail committed:
fixed map prototype for rolls

JasperStingtail committed:
minor changes, maints

DEATHB4DEFEAT commented:
Isn't that the Arborum park almost 1:1?

JasperStingtail commented:
Its not park station without a park. And its not exactly 1:1, just kinda did whatever

DEATHB4DEFEAT commented:
Man I forgot about how awful the tests are on this repo..

JasperStingtail committed:
arrivals changes, fixed an air alarm in epi

DEATHB4DEFEAT commented:
Disclaimer: I am not a good mapper, @Pspritechologist needs to review this :)



<details><summary><h2>Overall</h2></summary>
<p>

The meteor shielding is bad

> - Way too much of it
> - Too close to the station walls in many areas, it'll explode the walls still, just less
> - Looks awful
> - Shouldn't *all* be so randomly destroyed

Maintenance is alright

> - Too many straights / Lack of open or not super boring areas
> - Too much catwalk

---
</p>
</details> 



<details><summary><h2>Hallways</h2></summary>
<p>

The UnderVend being in the middle of a hallway is weird, move this and the ClothesMate to a bathroom
Edit: There's an UnderVend already in the bathroom, what's that other one doing in the hall?

![Content Client_Q4tPLJC9yv](https://github.com/Simple-Station/Parkstation/assets/77995199/423bc751-3e17-47a8-a206-c2fc0f877e28)

The Park is lacking in detail
Needs decals on the path
Put some more flowers and make the existing flowers not snapped to the grid

![Content Client_WQ3al0q1Ct](https://github.com/Simple-Station/Parkstation/assets/77995199/b95badf8-5bb0-45d9-bc97-45079e8f13c8)

Why is there a Smile outside Engineering?

![Content Client_mcSgb6Z1c5](https://github.com/Simple-Station/Parkstation/assets/77995199/91edadbd-41f0-4e8a-8b89-c7d589021731)

No comment

![Content Client_1f5rJAjbge](https://github.com/Simple-Station/Parkstation/assets/77995199/ebf90052-064c-4f63-a090-9c1dbb619675)

Why do these doors bolt each other?

![Content Client_08yOWsUC34](https://github.com/Simple-Station/Parkstation/assets/77995199/9fb38821-5f8b-46e2-a424-40356f9c0fbc)

Comment in image

![Content Client_t8dNSK60eq](https://github.com/Simple-Station/Parkstation/assets/77995199/66039432-e14a-44ae-bd8d-52fd33a34fc5)

Needs mirrors

![Content Client_MuEl4paeKh](https://github.com/Simple-Station/Parkstation/assets/77995199/995b71b8-9159-4b0d-8ebd-7976714ee437)

Not your fault but I want to comment on how stupid bookshelves look now

![Content Client_Itxe9lE34W](https://github.com/Simple-Station/Parkstation/assets/77995199/30370ce0-bf5a-468a-bf02-7a547396d367)

Shift the benches a tile south and move the front row west a tile or have them back to back

![Content Client_FdmuX6xETy](https://github.com/Simple-Station/Parkstation/assets/77995199/5bf24dc1-4f5c-4628-9865-37fda1f24b2b)

Arrivals feels a bit empty
Why only one dock per side? I think the ferry has two per

![Content Client_LO3BGNplN4](https://github.com/Simple-Station/Parkstation/assets/77995199/b81dc4c3-6495-4d67-82da-21fd5df6d845)

Maintenance is a hallway, right?
Disposals is pretty empty, make it looks more trashy and put some Janitor equipment around

![Content Client_X4u7NFZn5L](https://github.com/Simple-Station/Parkstation/assets/77995199/c5fe56fc-ff1d-4f73-8001-1f331ca98690)

---
</p>
</details> 



<details><summary><h2>Engineering</h2></summary>
<p>

I don't really like the sAI not having a satellite but this is a pretty nice room
The servers all need color decals

![Content Client_IauzsT4DCq](https://github.com/Simple-Station/Parkstation/assets/77995199/79ab28a2-2de6-40ca-a57a-85fcaed998c1)

Unrelated to the review, I hate this sign sprite

![Content Client_rlA0lMneS7](https://github.com/Simple-Station/Parkstation/assets/77995199/cde9bc96-1030-4510-a9a7-ec80f564bd88)

The AME being just right in the middle of everything is odd and should probably be more secure
Again, that massive sign sprite looks bad
2 starting fuel jar crates is too many
You have a parts crate, start the AME not constructed?
There should be a power monitoring console to see how much the AME is outputting, instead of staring at the SMES' to know if you're overpowering it a lot

![Content Client_CnTFw1W400](https://github.com/Simple-Station/Parkstation/assets/77995199/7f92060d-f70e-44db-8ec0-e163eef99071)

I like the air**lock** hallway into Atmospherics

![Content Client_spm5KIoVgx](https://github.com/Simple-Station/Parkstation/assets/77995199/9fcf5b97-0fc8-42e1-9df2-d4d0e57658f6)

This Solars lattice is way too fat
The panel extrusions are awkward
The meteor guards are too close, plentiful, and weirdly destroyed

![Content Client_nqRCnvV1cb](https://github.com/Simple-Station/Parkstation/assets/77995199/17dcfaa4-855c-4a29-90ee-73f8f4d59d8d)

I like Robotics

![Content Client_zLRKPvup78](https://github.com/Simple-Station/Parkstation/assets/77995199/dfd74006-8a17-44f9-a42a-c204f3b728df)

CE's office is also pretty nice, I like the space dragon figurine (and the potential reference to the dragon statue), didn't know that existed

![Content Client_yYeuSd30Zk](https://github.com/Simple-Station/Parkstation/assets/77995199/73e69587-c454-4573-844b-e4111bd54ee5)

This is an interesting layout
The board room should *maybe* use reinforced walls, it's not *that* valuable though..

![Content Client_hDFgG2nzSS](https://github.com/Simple-Station/Parkstation/assets/77995199/82eb3b80-899f-49b3-b055-29f31f24475d)

Back to Atmospherics
These signs still look bad but their *size* makes sense for where they are

![Content Client_5Mqs2HMWzV](https://github.com/Simple-Station/Parkstation/assets/77995199/3eeea178-31de-46df-bdbc-e00d6316da05)

This entire area is a lovely spaghetti of pipes
The meteor shielding that you can barely see are again too close and weirdly placed

![Content Client_7DO1283WEB](https://github.com/Simple-Station/Parkstation/assets/77995199/52a71bd1-14c8-4de5-baf9-3ab61ccccd4d)


---
</p>
</details> 



<details><summary><h2>Service</h2></summary>
<p>

<details><summary><h3>Cargo</h3></summary>
<p>

What the hell

![Content Client_2W3Do6mdFH](https://github.com/Simple-Station/Parkstation/assets/77995199/038d46d6-9859-49aa-bc6e-bc2180ee001a)

No shuttle :(

![Content Client_UpwRp6tLAD](https://github.com/Simple-Station/Parkstation/assets/77995199/49598748-eef3-4829-a02b-e85a614d0436)

I entered through maintenance, so I saw this last
Very cool

![Content Client_s3IR3XzdeK](https://github.com/Simple-Station/Parkstation/assets/77995199/3d625e1e-d0f1-43b7-a9bd-fd327bf9ad2e)

---
</p>
</details> 

The rooms don't feel themed enough
This is very cramped
Why are the supplies for Mime/Clown in an intermediate room if they have their own private rooms?

![Content Client_CSp6pOUbzy](https://github.com/Simple-Station/Parkstation/assets/77995199/ac796401-f5ca-4390-b302-a0ca99139f3e)

These spawners should be *on* the chairs

![Content Client_Dvep7ItFui](https://github.com/Simple-Station/Parkstation/assets/77995199/e27eed9d-c31d-40fe-8bf3-c2c3e4066c0a)

That's a lot of extra supply, you've got a booze dispenser and a booze vendor
Why are there two Bartenders? Or the supplies for two

![Content Client_PYDzzCJEms](https://github.com/Simple-Station/Parkstation/assets/77995199/30690047-b07d-42c8-91a8-b163b836dac5)

Boxers *and* Martial Artists!
The arena should have only a single tile open if any
The arena could be merged into the bar area, have the musician stage right by a slightly larger Boxing arena with more seats, battle music!
Needs a normal or brute medkit on the table

![Content Client_qSuvn5oM5I](https://github.com/Simple-Station/Parkstation/assets/77995199/6565a276-4020-4a61-b301-be6d5f9cf978)

...These crates really need to be not airtight or just don't use them at all

![Content Client_lylhymldLA](https://github.com/Simple-Station/Parkstation/assets/77995199/c59018ca-1bc1-42ed-8946-83240578c86e)

Kitchen's pretty nice, I like there being hot and cold food carts

![Content Client_915fMsMh2q](https://github.com/Simple-Station/Parkstation/assets/77995199/244089df-8f3b-4303-80ce-27d3041844ed)

No comment

![Content Client_5RVWxhRN22](https://github.com/Simple-Station/Parkstation/assets/77995199/120d2242-659c-472a-b322-c907e072a1e1)

---
</p>
</details> 



<details><summary><h2>Security</h2></summary>
<p>

This should use door bolts

![Content Client_a0OZBFjLeG](https://github.com/Simple-Station/Parkstation/assets/77995199/12edd3cf-4350-4897-98b3-7546f06438d1)

This hall is very blue

![Content Client_KuYrQoBqpQ](https://github.com/Simple-Station/Parkstation/assets/77995199/c28592ec-6fbe-45cc-b468-684b2a4072e8)

Need Prisoner clothing lockers in the cells

![Content Client_rbTMud1ERQ](https://github.com/Simple-Station/Parkstation/assets/77995199/ecdb5e26-857f-4050-9c33-da9ee6b18b0f)

This is an odd room

![Content Client_FUGv7pQpXE](https://github.com/Simple-Station/Parkstation/assets/77995199/75717404-c9dd-4c1b-bfc8-a9f01e5be23c)

🍕

![Content Client_pujRBCeknO](https://github.com/Simple-Station/Parkstation/assets/77995199/a3ce265a-b562-4802-ad35-4978474a3d37)

I like this perma, but it seems a bit odd how far it is from the Department and how there isn't a guideline going toward this and the courtroom

![Content Client_ozMIsuWs8m](https://github.com/Simple-Station/Parkstation/assets/77995199/ac1f01b7-b34c-4dcf-9fe3-171df50f9501)

---
</p>
</details> 



<details><summary><h2>Medical</h2></summary>
<p>

I entered through the backside unintentionally and wanted to find the entrance -- this Medical is *huge*

Very nice reception area, love the kids' area
How would benches look instead of chairs?
The kids' area should have some separator instead of a sudden carpeted area

![Content Client_j8uX4xMCQm](https://github.com/Simple-Station/Parkstation/assets/77995199/2b1e9312-3f8d-4463-a00e-daddbcda2385)

I was expecting this to bolt

![Content Client_KORQrLEAsW](https://github.com/Simple-Station/Parkstation/assets/77995199/80be15ea-c264-4adb-9cd7-fc25840d7934)

No comment

![Content Client_s6ygRVBw7s](https://github.com/Simple-Station/Parkstation/assets/77995199/3fd6889b-c200-4d3d-a0f4-f235da7396dd)

Chemistry is pretty nice

![Content Client_GQdv9WKH59](https://github.com/Simple-Station/Parkstation/assets/77995199/82414969-afbc-4ab9-81f4-6ab1d701178c)

These three wall lockers are unreachable and weirdly positioned (not anchored to the grid?)

![Content Client_O9Jzrjb8WU](https://github.com/Simple-Station/Parkstation/assets/77995199/8b3e377e-f3f8-41f2-be14-125bf5ab2c9c)

Virology bolts :)
That extra door without an intermediate room is a bit odd

![Content Client_j1AuM3e0ZY](https://github.com/Simple-Station/Parkstation/assets/77995199/22598bbe-27ce-44ca-89b0-1e9f060f7155)

Cloning's layout seems a bit odd, and should have some privacy for the cloned person
Why are the shirts across the room?

![Content Client_ZDQ6UDahZd](https://github.com/Simple-Station/Parkstation/assets/77995199/9c9d2443-2109-4e76-b6bd-147c636344f5)

Needs a full surgery kit
There should be privacy shutters, people may not want an audience or want to *be* the audience

![Content Client_6GN615yVnD](https://github.com/Simple-Station/Parkstation/assets/77995199/93f72b19-3f36-4bbd-ae43-aaf99a34882a)

Psychologist office needs privacy shutters

![Content Client_Hy78XGHpIl](https://github.com/Simple-Station/Parkstation/assets/77995199/aa0c7594-49c3-4414-939a-aea3581a6a2e)

The Smile keeps following me just barely out of frame of any screenshots

---
</p>
</details>



<details><summary><h2>Command</h2></summary>
<p>

Why is the Vault so large and empty?

![Content Client_l5jxUGYJEO](https://github.com/Simple-Station/Parkstation/assets/77995199/9b148d5a-454b-4763-a13a-ec2de804d611)

:)

![Content Client_tLqBcBRAzN](https://github.com/Simple-Station/Parkstation/assets/77995199/8ffa6909-a517-460f-9b23-8a60935330ce)

:)

![Content Client_KP2b6ikhB0](https://github.com/Simple-Station/Parkstation/assets/77995199/17bdb6e9-6e01-4a0e-b8f1-3c48cd80f6bb)

Command doesn't feel much like the "head" of the station
It looks pretty uninteresting and empty
The mess of meteor shielding outside looks bad

![Content Client_P9JgSHBQoA](https://github.com/Simple-Station/Parkstation/assets/77995199/c324d608-c8ed-4bd8-8739-e9b9291ec726)

---
</p>
</details>



<details><summary><h2>Epistemics</h2></summary>
<p>

Lots of empty space here

![Content Client_Fzv6wSpaTW](https://github.com/Simple-Station/Parkstation/assets/77995199/09fcca9e-e8fc-4f75-8cd9-748ee96c6f2f)

This area is way too cramped with how much empty space it has
The cycling room also doesn't bolt
No predefined space for expansion of glimmer devices

![Content Client_QwDIsAYkYI](https://github.com/Simple-Station/Parkstation/assets/77995199/2d006249-bec5-4d86-ac8a-0bce8f2d144c)

Replace the Atmospherics Control Chamber glass with normal plasmaglass
Replace the locker room's plasmaglass with normal or reinforced glass
The locker room has a lot of empty space
Why are there 2 golem shells in maintenance?

![Content Client_3lttfT0Fez](https://github.com/Simple-Station/Parkstation/assets/77995199/f5edc0cf-abd0-415d-b183-a9b1e83929d6)

The Chapel is pretty nice

![Content Client_GpTQ0ZFjWk](https://github.com/Simple-Station/Parkstation/assets/77995199/20f3cb03-673b-46b3-b705-107cc4d3b711)

---
</p>
</details>


![Content Client_EBo7xXsqyL](https://github.com/Simple-Station/Parkstation/assets/77995199/2343536b-2b22-4eda-86d8-bef007ca622d)

DEATHB4DEFEAT reviewed (changes requested):
I should've said my comment as a review.

JasperStingtail commented:
# Overall
 Meteor shield improved, actually a meteor shield now. Minor maintenance improvements.
# Hallways
Fixed undervends.
Added park decor.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/0056ea74-d10f-448c-a3c1-ba3dfdbc616f)
Smile slime moved to epi hallway.
Removed bolts in grav room.
Accessible toilet paper! 
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/ed0d4258-8b62-4efd-85f8-b287839b9da8)
Mirrors (i dont like how they get forcefully orientated, but alright.)
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/1caf29ce-3b5f-491e-b96e-11ebfcf48b70)
Bench.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/dcbfb952-6601-429f-91ae-8b304d47759d)
Added plants in arrivals.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/2c945fe2-e0ed-4161-afe6-c3c85553186d)
Disposals has been rearranged.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/05f55c91-4255-4cb2-a9c6-5eb22f05a775)
# Engineering
Most decals are covered by the camera router entities themselves and all other available ones look ugly, best left without them.

I dont really think the AME is in the middle of everything, but now is deconstructed and has a power monitoring console.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/3674d86b-87d7-4833-b32a-beb2b180aade)

Solars meteor shielding is better. Meteors are highly destructive and if too many hit during one event its just automatic shuttle call.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/91f05c54-d30d-452c-90d9-cbb61e9ab531)

Issue/question with robotics: engineering personnel dont have access to the Robotech Deluxe. Is this supposed to be epi?
Board room compromise: hallway has reinforced wall, but not the inside.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/bf6d3bdb-aaa3-490b-8ea0-d9716796c578)

# Service
Its small because its really a minor room/storage room really. Mime and clown have separate rooms for specialized bedrooms.

I think the station is sizeable enough to handle two bartenders.

On the arena having a musician area: the wall currently dividing them has important devices on it- APC, fire alarm, fire extinguisher- and re-working this may lead to even more odd placement of those items, so I added the smallest musician area possible.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/8833f62c-0bde-4ff1-98c0-33a76c312604)

Airtight crates fixed (removed).

# Security
Those doors now bolt.

How about this?
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/21c54571-9ee8-4c3e-9c1f-c5a0ea59167f)

Prisoners have wardrobes now.

Perma is far away so that security can have a bit more responsibility over their department rather than staying in a room all shift. Also, the possibility of prison guards as spawnable roles? Their level of access is equal to that of a sec officer (security) but they dont have perma (armory) access. Should perma be changed to security officer access?
Also, it has decals now.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/d1d7d418-5019-48a0-b94b-d1a02414496d)

# Medical
That's how benches look. Also, decals on the kids carpet; good for a separator?
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/0f55be60-c96c-473d-a8f5-342f9e876955)

That extra door set has been removed.

Replaced the wall lockers with medicine ones (the wardrobe closets acted like standalone lockers, rather than wall lockers).

![image](https://github.com/Simple-Station/Parkstation/assets/130887787/105bdd95-766f-4eb6-99f2-ec53d631b5e0)
About this: this is to mainly separate the injured from the sick, not separating people/infections. The hallway to the south should be used almost exclusively by medical personnel, so if a doctor didn't take proper precautions after treating the sick, that's an IC issue.

Cloning has been somewhat fixed, but I have a small idea for reducing it oddness after the rebase.

Surgery kit added.

Phycologist has privacy shutters now.

# Command
I dont really know, I'll get back the vault at some point.

Command figured out how to turn on dark mode.
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/14a6d2f9-00ae-4d2d-9095-935b1befa961)

# Epistemics
I have a small idea, not sure if I like it yet, but I think thats the best option for epi lockeroom for now.

Anomaly room got even more cramped when I tried rearranging things.

Interchange bolts now.

Glass got replaced as described. The extra golems is maints are exactly that: spares.

Appendix :)

DEATHB4DEFEAT commented:
> Issue/question with robotics: engineering personnel dont have access to the Robotech Deluxe. Is this supposed to be epi?

Robotics is supposed to be Science upstream, we need to fix whatever's locked since it wasn't before

JasperStingtail commented:
I had forgotten I did this actually.

![image](https://github.com/Simple-Station/Parkstation/assets/130887787/22692230-e239-499c-a772-ea8e11f7e17f)

ThatFunkyGuy commented:
Iz very nice map, I enjoy great. please add more scary apple ☺☺☺☺ please port to 13 thank you! is nice. 
![channels4_profile (1)](https://github.com/Simple-Station/Parkstation/assets/155513847/97500fa7-64ad-4926-bfc8-acbdb2b99777)

JasperStingtail commented:
https://pikmin.wiki.gallery/images/thumb/3/37/Pik_yellow.gif/240px-Pik_yellow.gif

DEATHB4DEFEAT commented:
Who's this funky guy?

JasperStingtail committed:
PR requested changes

JasperStingtail commented:
i love grids i love floating items when theyre not supposed to be

JasperStingtail committed:
fixed a light on the wrong grid, aesthetic meteor shield change

JasperStingtail committed:
this is why we cant have nice things

JasperStingtail commented:
Hooked up the telepad in cargo. Wizden moment

JasperStingtail committed:
improved the vault, fixed trash cans

JasperStingtail committed:
grids, grids aoaooaoa

ThatFunkyGuy commented:
why not merged yet. am very angry 😡😡😡 I still wait for map to be on 13. make it NOW.

JasperStingtail commented:
me looking at you
![image](https://github.com/Simple-Station/Parkstation/assets/130887787/024d1ad3-e6d8-4683-8ccd-1ff68259278b)

JasperStingtail committed:
fixed a couple fire alarms

JasperStingtail commented:
Emergency lights were weird, also fire alarm moment

JasperStingtail committed:
lots of changes

JasperStingtail committed:
more work, saving progress

JasperStingtail committed:
flashes are unanchored now

Pspritechologist commented on Resources/Prototypes/Maps/gusstation.yml:
```diff
@@ -0,0 +1,57 @@
+          mapNameTemplate: '{0} Aegis {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'TG'
```
This can be any custom set of letters (recommended to be 2-3 letters long) and is intended, I believe, to be the constructor of the station. It is read in game something like 'NT-TG' or just 'NTTG'. Not 100% sure.

JasperStingtail replied:
What does this mean? Are you talking about line 14?

Pspritechologist replied:
Yeah, line 14 is the one the comment is placed on but GitHub doesn't make that obvious. 'prefixCreator' modifies the in-game display name of the station, since the final name will be something like 'NTTG Aegis xx###' something like that.
My point is you can change it to represent the constructor of the station. TG is just what was used for the station I copied the file from for Park originally :P

JasperStingtail committed:
more changes from review, almost done

JasperStingtail committed:
finished map review changes

JasperStingtail commented:
Wooo yeaa oh yea wooooo

JasperStingtail commented:
Also, the medical doctor's wall locker [LockerWallMedicalDoctorFilled] needs to be an actual wall locker than a regular locker.

JasperStingtail committed:
cameras

JasperStingtail committed:
brig timer change
